docs/interface/dashboard.md

```diff
@@ -1,30 +1,16 @@
 # Course dashboard
 
-![Teach dashboard](/images/courses/teach-dashboard.png)
-*Teach dashboard*
+![Teach dashboard](/images/interface/content-dashboard.png)
+*Content dashboard*
 
-The landing page consists of two parts: a sortable, filterable and paginated table of course repositories and a modal to create new courses. The table has the following columns:
+The content overview page consists of two parts: a filterable and paginated table of created content and a modal to create new or import existing courses. The table has the following columns:
 
 - **Title**: the title of the course (or repository name if it is not properly built yet)
-- **Owner**: the owner of the repository. If the repository has no owner yet, you can claim it if it's yours.
+- **Status**: current state of the content (development, live)
 - **Instructors**: instructors of the repository. You can set these in the Settings tab of the master branch of the repository.
-- **Updated by me**: when the repository was last updated by me
-- **Updated by instructor**: when the repository was last updated by an instructor
-- **Un reviewed time**: the time differences between when the repository was last updated by me and when it was last updated by an instructor
-- **Last updated branch**: the branch of this repository that was last updated
+- **Last edited**: time when the last update has been done to the content
+- **Edit action**: opens editor for editing the content
 
-For featured courses the following extra columns are in the table:
+For premium content some columns are redefined:
```

Comment:
"redefined" from what?

Reply:
I don't think we should even be saying what it used to be. Let's just tell them what it is now.